tmail-backend/jmap/extensions-rabbitmq/src/main/java/com/linagora/tmail/james/jmap/RabbitMQEmailAddressContactSubscriber.java

```diff
@@ -87,12 +119,24 @@ private Optional<EmailAddressContactMessage> toMessage(Delivery delivery) {
         }
     }
 
-    private Mono<Void> dispatchEvent(TmailEvent tmailEvent) {
-        return eventBus.dispatch(tmailEvent, EVENT_REGISTRATION_KEY);
+    public Flux<EmailAddressContactMessage> receivedMessages() {
+        return listener.asFlux();
+    }
+
+    private Disposable messageConsume() {
+        return receivedMessages()
+            .flatMap(message -> Mono.from(messageHandler.handler(message))
+                .onErrorResume(error -> {
+                    LOGGER.error("Error when consume message '{}'", message, error);
+                    return Mono.empty();
```

Comment:
Should we ignore the error when consuming messages?

Reply:
If it fails it's mainly because of a deserialization issue? I guess we can ignore, we probably don't want to requeue messages if they are not well serialized? But maybe I'm wrong

Reply:
> If it fails it's mainly because of a deserialization issue?
No, the plain deserialize will be done in the previous step
Here comes 2 reasons: 
- Fail when parsing to Username or Domain 
- error from ES indexing

Reply:
That is a good question.

We should nack without requeue. If deadlettering is in place the message will then be moved to dead letter in face of errors. Could later be audited and recovered.

(Do we have deadletter in place?)